src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -80,9 +88,13 @@ def __init__(
     ) -> None:
         super().__init__()
 
-        self.task = task
+        self.task = tabular.TabularPredictor
         self.freq = freq
         self.prediction_length = prediction_length
+        if ag_path_prefix is not None:
+            self.path = ag_path_prefix / self.ag_path
+        else:
+            self.path = self.ag_path
```

Comment:
I’m wondering: why not take `ag_path` directly as input? With default value `”ag_tabular_models”`. That way these four lines can be replaced by `self.ag_path = ag_path`

Reply:
I no longer ask the user to enter a path when estimator is created. See the comments below.